### DISCUSSION TITLE
French Language Leader

### DISCUSSION THREAD

**Discussion by benjaminlecouteux** [OHF-Voice/intents#359] [Language Leaders]
Hello, if the lead is available, I would be interested in contributing to the implementation/validation of ASR French grammars. I have some experience in this field and would like to contribute contribute to HA.

**Comment by synesthesiam** [comment]
@flexy2dd, @acesyde, @piitaya - moved language leader discussion here.

**Comment by synesthesiam** [comment]
@benjaminlecouteux Are you still interested in being a language leader?

**Reply by benjaminlecouteux** [reply]
Yes, of course.

**Reply by balloob** [reply]
Can you please accept the invitation for the Language leader team on GitHub?

And please keep an eye out for French PRs. Once you accepted the invite you will receive GitHub notifications. As a LL you're the gatekeeper for contributions to the French language and we wouldn't want it to fall behind, especially now that interest in contributing is high.

**Reply by flexy2dd** [reply]
Yes of course

**Comment by synesthesiam** [comment]
There are many open French PRs; is anyone else able to help?

**Comment by piitaya** [comment]
I will look at this ☺️

**Comment by acesyde** [comment]
We are on it 👍

**Comment by acesyde** [comment]
@slegars56 @Gorouk @baptistebisson  je pense qu'il faut se synchroniser sur vos PR sinon ça va être compliqué de merger :)

**Reply by Gorouk** [reply]
@acesyde ah top, je n'avais pas réussi à mettre la main sur cette discussion (git noob que je suis...)
@slegars56 @baptistebisson comment vous voulez qu'on fasse ?

**Reply by baptistebisson** [reply]
J'ai mal fait les choses car je n'ai pas vérifié les autres PR, désolé. 
Je propose de s'organiser sur le plus ancien PR ou alors sur celui avec le plus de modifications.

**Reply by acesyde** [reply]
@baptistebisson excellente initiative, prenez en compte les remarques de @piitaya dans la PR #466

**Reply by piitaya** [reply]
On peut se concentrer sur les phrases et les tests. On pourra ensuite améliorer les "expansion rules" pour éviter les duplications (futurs PRs). Un refactoring trop tôt peut nous limiter dans la definition des phrases.

**Comment by synesthesiam** [comment]
@CedricFinance @piitaya @flexy2dd @baptistebisson - any one interested in being a language leader for French? We're hoping to get response translations in soon.

**Reply by flexy2dd** [reply]
Yes for sure

**Reply by synesthesiam** [reply]
Great, I'll add you. Thanks!

**Reply by piitaya** [reply]
Opened a PR : https://github.com/home-assistant/intents/pull/744

**Comment by synesthesiam** [comment]
@flexy2dd when you get a chance, please take a look at the translations for `HassGetState`

**Reply by flexy2dd** [reply]
Yes sure I'm on it

**Reply by flexy2dd** [reply]
Done

**Reply by synesthesiam** [reply]
Thanks @flexy2dd!

**Comment by synesthesiam** [comment]
@benjaminlecouteux Since you haven't been active for a while, I'm going to remove you as a language leader. If you have time again in the future, let me know.
